card/db/schema.rb

```diff
@@ -38,7 +38,7 @@
   create_table "card_changes", force: :cascade do |t|
     t.integer "card_action_id", limit: 4
     t.integer "field",          limit: 4
-    t.text    "value",          limit: 65535
+    t.text    "value",          limit: 16777215
```

Comment:
I'd tend towards no limit unless there is a benefit. The benefit (or lack) may depend on which db you are using, you could get different underlying types with different behavior, but generally we want the largest it will give us.

We had an extended debate and comparison of the performance of different scenarios.  This was all for Oracle, but it could be like this or worse on others.  Bottom line is the *LOB types are often stored very differently and it will take several round trips to the DB if your query returns one of these.

To summarize, you could potentially get some performance gain by having a split field where it used a type that fits, plus a second column for large content. So, you could query just the short column in general, but make a different query when you need to return big values too.

All the complexity probably isn't worth it, so just go with the biggest possible.